tests/test_slearner.py

```diff
@@ -72,11 +76,16 @@ def test_append_treatment_to_covariates(
         )
         assert (X_with_w[:, :-3] != covariates).sum() < 1
     else:
-        assert isinstance(X_with_w, pd.DataFrame)
+        if backend == "pd":
+            assert isinstance(X_with_w, pd.DataFrame)
+        elif backend == "pl":
+            assert isinstance(X_with_w, pd.DataFrame)
```

Comment:
should this be `pl`?

Reply:
It shouldn't have been but after reworking the way we handle treatment assignments in the S-Learner(https://github.com/Quantco/metalearners/pull/105) it now is expected to be pl`. :)